### ISSUE TITLE
Search with no results found, doesn't return any visual feedback

### ISSUE THREAD

**Issue by Mehradzie** [vimperator/vimperator-labs#30]
This might not be a big issue however, it will be good to have this feature. When you press "/" and start typing for a keyword to search in the page, the search follows your typing and finds the matches and when it hits the point which it fails (meaning the word you're looking for doesn't exist in the page, it sounds a ding) there is no visual feedback to know if your search process finished with 0 results or maybe your search result is highlighted somewhere in the current page that you can't see. (possible confusion, you see!)

![vimperatorfeedback](https://cloud.githubusercontent.com/assets/7462584/4054289/cbb8b986-2d8c-11e4-9e06-b9a825919e76.jpg)

Specially in my case I mute my computer at the office and I get caught in this situation everyday since the only feedback for the 0 result search is a voice unless you press enter and then "N" to see the visual red ribbon on the bottom stating your search didn't return any results. 
Since this would be a relatively easy improvement, I would like to submit this request. 

Thanks everybody.

**Comment by gkatsev** [comment]
I believe this has been fixed by #93 for some definition of "fixed".

**Comment by Mehradzie** [comment]
I honestly don't think the UI is responding to this issue as it should. You
still need to hit Enter after you typed the whole search phrase.
The matching can be done on the air while each character is being typed.

**Comment by gkatsev** [comment]
Yeah, though, with latest vimeprator, I end up seeing the native find bar show up for some reason when I end up typing something that no longer exists, so, clearly, it's trying to do the right thing.
There's, unfortunately, more work to be done here. Reopning.

**Comment by Mehradzie** [comment]
This is a worthy feature to have, especially for those who are using Vimperator in the day to day use.

Thanks for reopening.
